### ISSUE TITLE
Can't select new data package

### ISSUE THREAD

**Issue by DenoBeno** [clarity-h2020/docker-drupal#100]
I have just tested this on https://csis.myclimateservice.eu/study/17/view/data - can't choose the data package for the study.

**Comment by DenoBeno** [comment]
Any idea what could be causing this? Is it the new theme?

**Comment by fgeyer16** [comment]
I looked at this and found that the user misses easily a step because he does not see it.
We have this "use data package" button in the entity browser. This has to be clicked to confirm the selection of datapackage.
![change_datapackage1](https://user-images.githubusercontent.com/25331450/63698228-b8696600-c81e-11e9-8912-d4eb076f6ebb.png)
If we select a data package some infos about it will be shown below the list of datapackages. Doing so the button moves down and out of sight. Without clicking this button we will have an empty selection. And so it seems not to work.
![change_datapackage2](https://user-images.githubusercontent.com/25331450/63698426-20b84780-c81f-11e9-9b5f-89281a747b69.png)
So I moved up the footer of the entitybrowser view in which this button will be inserted by the entity browser module.
![change_datapackage3](https://user-images.githubusercontent.com/25331450/63698664-97eddb80-c81f-11e9-89de-c40af87306c1.png)
Now the button hopefully will not missed that easily. 
This is done by css `flex-flow: column-reverse;`property in css. This orders html elements in the revers order of their order in the dom. The drawback is that now the view footer is above the view and the view header below.
I documented this in the administrative comment of the display of the view (section advanced)
